### DISCUSSION TITLE
Volume control on speakers connected to a HP u28

### DISCUSSION THREAD

**Discussion by manu-del** [MonitorControl/MonitorControl#821] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Volume control is not working on my speakers, connected to my monitor HP u28.
I can change the level on the app but the sound doesn't change. Is there a way to do make it work?

### Expected behavior

I want to control the volume level of my speakers with the volume keys on my keyboard. The speakers are connected to my monitor through mini jack, and my monitor is connected to my macbook pro through usb-c.

### Environment Information (please complete the following information)

```markdown
- macOS version: Monterey
- Mac model: Macbook pro 15" 2016
- MonitorControl version: 4.0.2
- Monitor(s): HP u28
- Monitor Cable(s)/Connection(s): usb-C
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
The display probably does not process the DDC volume command. Some displays might require a special speaker select command to choose from the appropriate output, but it is basically impossible to test and figure out without hands-on with the display. You might want to try to experiment with various volume related commands via [ddcctl](https://github.com/kfix/ddcctl). If you find a combination that works, a feature request can be added here to support that specific combination of commands.

I'll convert this into a discussion so others with similar displays may add their voice.
